src/core_plugins/kibana/public/visualize/wizard/wizard.js

```diff
@@ -29,9 +27,15 @@ module.controller('VisualizeWizardStep1', function ($scope, $route, kbnUrl, time
   kbnUrl.removeParam(DashboardConstants.ADD_VISUALIZATION_TO_DASHBOARD_MODE_PARAM);
 
   $scope.visTypes = Private(RegistryVisTypesProvider);
+
   $scope.visTypeUrl = function (visType) {
-    const baseUrl = visType.requiresSearch ? '#/visualize/step/2?' : '#/visualize/create?';
+    const baseUrl =
```

Comment:
* `#/visualize/step/1` => `/visualize/new`
* `#/visualize/step/2` => `/visualize/new/configure`

I'll add redirects here. Any suggestions on the best way to do that?